plugins/PRAplugin/doc/include/MCSsolver.tex

```diff
@@ -0,0 +1,74 @@
+\section{MCSSolver}
+\label{sec:MCSSolver}
+
+This model is designed to read from file a list of Minimal Cut Sets (MCSs) and to import such Boolean logic structure as a RAVEN model.
+Provided the sampled values of Basic Events (BEs) probabilities, the MCSSolver determines the probability of Top Event (TE), i.e., the union of the MCSs.
+The list of MCS must be provided through a CSV file with the following format:
+
+\begin{table}
+  \begin{center}
+    \caption{MCS file format.}
+    \label{tab:table1}
+    \begin{tabular}{c|c|c} 
+      \textbf{ID} & \textbf{Prob} & \textbf{MCS}\\
+      \hline
+      1, & 0.01, & BE1\\
+      2, & 0.02, & BE3\\
+      3, & 0.03, & BE2,BE4\\
+    \end{tabular}
+  \end{center}
+\end{table}
+
+In this example:
+\begin{itemize}
+  \item three MCSs are defined: MCS1 = BE1, MCS2 = BE3 and MCS3 = BE2 and BE4 
+  \item four BEs are defined: BE1, BE2, BE3 and BE4
+  \item probability of TE, i.e. P(TE), is equal to: $P(TE) = P(MCS1 \cup MCS2 \cup MCS3)$
+\end{itemize}
+
+Note that the MCSSolver considers only the list of MCSs and it discards the rest of data contained in the csv file.
+
+All the specifications of the MCSSolver model are given in the \xmlNode{ExternalModel} block. 
+Inside the \xmlNode{ExternalModel} block, the XML nodes that belong to this models are:
+\begin{itemize}
+  \item  \xmlNode{variables}, \xmlDesc{string, required parameter}, a list containing the names of both the input and output variables of the model
+  \item  \xmlNode{solverOrder},\xmlDesc{integer, required parameter}, solver order for $P(TE)$ 
```

Comment:
I think you need to provide a little bit more information regarding the solverOrder